docs/virtualkeyboard/index.html

```diff
@@ -0,0 +1,492 @@
+                <li>
+                When <dfn>empty string</dfn> enumerated attribute is set, the VK is handled by the browser.
+                </li>
+            </ul>
+        </p>
+        
+    </section>
+    <section>
+        <h2>
+          Virtual Keyboard Visibility Change CSS environment variables.
+        </h2>
+        <p>
+          The {{VirtualKeyboard}} API proposes six new <a href="https://drafts.csswg.org/css-env-1/#environment">CSS environment variables</a> that the web developers can utilize to calculate the virtual keyboard's size and adjust layout in a declarative way.
+        </p>
+        <p class="note">
+          These CSS env should be added to the <a href="https://drafts.csswg.org/css-env-1/#env-function">CSS env variable spec</a>.
+        </p>
+        <section>
+            <h3>Keyboard inset variables</h3>
+            <table data-for="env()">
+                <tr>
+                    <th>Name
+                    <th>Value
+                <tr>
+                    <td><dfn>keyboard-inset-top</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+                <tr>
+                    <td><dfn>keyboard-inset-right</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+                <tr>
+                    <td><dfn>keyboard-inset-bottom</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+                <tr>
+                    <td><dfn>keyboard-inset-left</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+                <tr>
+                    <td><dfn>keyboard-inset-width</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+                <tr>
+                    <td><dfn>keyboard-inset-height</dfn>
+                    <td><a href="https://drafts.csswg.org/css-values-3/#length-valuehttps://drafts.csswg.org/css-values-3/#length-value"> length</a>
+            </table>
+            
+            <p>The keyboard insets are six <a href="https://drafts.csswg.org/css-env-1/#environment">environment variables</a> that define a rectangle by
+            its top, right, bottom, and left insets from the edge of the viewport. Default value of the keyboard insets are "0px" if a fallback value is not provided else it gets updated when {{VirtualKeyboard/boundingRect}} value changes.
+            The width & height insets are calculated from the remaining insets for developer ergonomics.
+            </p>
+        </section>
+    </section>
+    <section>
+        <h2>Examples</h2>
```

Comment:
I think there were [good images in the explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/VirtualKeyboardAPI/explainer.md#examples) to go with these examples.  Can you include them to help illustrate the impact of the code?

Reply:
added